css/main.css

```diff
@@ -325,3 +325,60 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+#contact-form {
+  padding: 10%;
+}
+
+#contact h2 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
```

Comment:
- [ ] The `id` name of this block seems to be incorrect 😔 
I guess you were trying to type `#contact-form` instead. Kindly check and fix this. Once this has been fixed, the heading will definitely look more like the one in the template. 🕺